test/message/throw_in_line_with_tabs.js

```diff
@@ -1,4 +1,25 @@
-/* eslint-disable indent, no-tabs */
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Oh, wait, I see, the copyright has not been restored to the v6.x-staging branch, at least not yet. OK, I'll remove it.